source/renderer/app/stores/CurrencyStore.ts

```diff
@@ -127,10 +128,23 @@ export default class CurrencyStore extends Store {
       this.getCurrencyRate();
       await this.api.localStorage.setCurrencySelected(selected.code);
     }
+
+    this.analytics.sendEvent(
+      EventCategories.SETTINGS,
+      'Changed currency',
+      code
+    );
   };
   @action
   _toggleCurrencyIsActive = () => {
     this.isActive = !this.isActive;
     this.api.localStorage.setCurrencyIsActive(this.isActive);
+
+    this.analytics.sendEvent(
+      EventCategories.SETTINGS,
+      `Turned ${
+        this.isActive ? 'on' : 'off'
```

Comment:
In this place you're doing ternary in string interpolation, in the other it's string or string (duplicating the text) 😅

I still recommend some constant/s for event names.

Reply:
Yes, that's something I overlooed. Fixed.